test/videojs-hls_test.js

```diff
@@ -1734,7 +1733,7 @@ test('calls ended() on the media source at the end of a playlist', function() {
   requests[0].response = new ArrayBuffer(17);
   requests.shift().respond(200, null, '');
 
-  strictEqual(endOfStreams, 1, 'ended media source');
+  strictEqual(updateendListener, 1, 'ended media source');
```

Comment:
`updateend` should fire after every appendBuffer() operation has completed, not the end of the stream. I think this test is asserting on the wrong thing now.